pkgs/development/python-modules/hdfs/default.nix

```
@@ -0,0 +1,34 @@
+{ buildPythonPackage
+, docopt
+, fastavro
+, fetchFromGitHub
+, lib
+, nose
+, pytestCheckHook
+, requests
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "hdfs";
+  # See https://github.com/mtth/hdfs/issues/176.
+  version = "2.5.8";
+
+  src = fetchFromGitHub {
+    owner = "mtth";
+    repo = pname;
+    rev = version;
+    hash = "sha256-94Q3IUoX1Cb+uRqvsfpVZJ1koJSx5cQ3/XpYJ0gkQNU=";
+  };
+
+  propagatedBuildInputs = [ docopt requests six ];
+
+  checkInputs = [ fastavro nose pytestCheckHook ];
+
+  meta = with lib; {
+    description = "HdfsCLI: API and command line interface for HDFS.";
```

```suggestion
    description = "Python API and command line interface for HDFS";
```

No period at the end -> see https://nixos.org/manual/nixpkgs/stable/#var-meta-description

If a heavy use case is the command line then it should also go to `all-packages.nix` with `toPythonApplication`.

---

Ah, thanks for that link! I wasn't aware of those style guidelines